source/develop/release-management/features/virt/enhance-import-export-with-ova.md

```diff
@@ -0,0 +1,135 @@
+
+# Scope
+This feature is about enhancing the import VM and export VM processes in oVirt by making a broader use of the OVA format.  
+
+## Upload OVA
+Today, the process of importing an OVA is limited to OVA files compatible with those produced by VMware and requires one to make the OVA file accessible by  one of the hosts managed by oVirt with the right file permissions (vdsm:kvm). It is a cumbersome and error-prone process.  
+
+This feature provides a broader and more user-friendly alternative that enables one to upload a local OVA file straight from a client (browser/rest-api based, much like the upload-image functionality that was recently added to oVirt).  
+
+It would be possible to upload OVA files that were generated either by oVirt or by external sources. Officially, the only external source being supported is VMware. However, in practice, the process could work with OVA files generated by other source as long as their OVF configuration contains the data oVirt consumes (in the expected form) and virt-v2v knows how to convert them.
+
+## Export as OVA
+Today, users are provided with two ways to migrate VMs and templates between environments. First, one can use an export domain. However, this approach is suitable only for migration between different oVirt environments. Moreover, there is a plan to drop the support for export domains. Second, one can detach a data domain and attach it to another setup. This way is tailored to disaster recovery and less fits the migration flow due to the side effects in detaching a data domain.  
+
+This feature provides an alternative way for a migration between environments by generating an OVA from an existing VM or template. By encapsulating all VM/template data in a single file, we achieve an improved portability. By being aligned with the common OVA specification (rather than our unique structure of a storage domain), we achieve better compatibility with other platforms.  
+
+The OVA file would be streamed into imageio proxy, enabling its creation on a variety of locations (path on host in the data-center, NFS share, etc). In addition, it will include neither snapshots nor the template that the VM may be based on.  
+
+Moreover, this feature enables to export a running VM with no downtime, unlike today that one is required to shutdown the VM in order to be able to export it.
+
+## Enhance Import OVA
+This feature enhances the current support provided for import OVA files accessible to the host by enabling to specify a path to OVA that was generated by oVirt. Import of OVA that was generated by oVirt does not involve the adaptation needed for OVA files generated by other platforms (e.g., installing kvm-specific drivers) and thus, should be faster than importing of such OVA files.  
+
+## Download OVA
```

Comment:
It is ok not to be 100% clear on this part - it is kind of TBD, we'll need to discuss it later when the storage team starts working on the download-image. I added some explanation though